src/components/Inputs/URLInput/URLInput.tsx

```diff
@@ -1,6 +1,6 @@
 import { FC } from 'react'
 
-import { TextInput } from '@/components/Inputs/TextInput'
+import { TextInput } from '@/components/Inputs/TextInput/TextInput'
```

Comment:
Prefer import from index
```suggestion
import { TextInput } from '@/components/Inputs/TextInput'
```